xdsl/interactive/mlir_opt_pass_screen.py

```diff
@@ -0,0 +1,44 @@
+from textual import on
+from textual.app import ComposeResult
+from textual.containers import Horizontal, ScrollableContainer
+from textual.screen import Screen
+from textual.widgets import Button, TextArea
+
+
+class MlirOptPassScreen(Screen[str]):
```

Comment:
Why the separate screen? Multiple of the passes accept arguments, it would be good to handle all of them, and ones that xDSL doesn't know about when used as a dependency.

Reply:
I think that sort of makes sense as a first step, it would be nice if the UI could actually present the user with the list of available arguments, and default values, but we don't have APIs for that yet. I think to start with it could just take a string and it would be up to the user to make sure that it works.